### PR TITLE
chore: udpate latest deps

### PR DESCRIPTION
**Pull request by Baha-sk**
Signed-off-by: Baha Shaaban <baha.shaaban@securekey.com>

### CODE REVIEW

**Review by fqutishat — CHANGES_REQUESTED** [review]
you need to update agent-sdk

**Comment by Baha-sk** [comment]
> you need to update agent-sdk

updated in web-wallet

File: cmd/wallet-web/src/pages/WACIShare.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
 -->
 
 <template>
-  <div class="w-full flex justify-center items-start">
+  <div class="flex justify-center items-start w-full">
```
Comment (rolsonquadras) [inline cmd/wallet-web/src/pages/WACIShare.vue:8]: was this updated automatically ?

Comment (rolsonquadras) [reply cmd/wallet-web/src/pages/WACIShare.vue:8]: If not, can we ignore this file from check-in ?

Comment (Baha-sk) [reply cmd/wallet-web/src/pages/WACIShare.vue:8]: yes

Comment (rolsonquadras) [reply cmd/wallet-web/src/pages/WACIShare.vue:8]: @birtony

**Review by fqutishat — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

**Review by sudeshrshetty — APPROVED** [review]

File: cmd/wallet-web/src/pages/WACIShare.vue (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/wallet-web/src/pages/WACIShare.vue:8]: ignore this file @Baha-sk

**Pull request merged**